### ISSUE TITLE
Overwriting a saved game in iOS 8.3 closes the story

### ISSUE THREAD

**Issue by GoogleCodeExporter** [ifrotz/iosfrotz#257]
```
What steps will reproduce the problem?
1. Load up any game. Save the game as you normally would.
2. Save the game again, overwriting your previous saved game.

*What is the expected output or behavior? 
To go back to the game and keep playing.

*What do you see instead?
iFrotz visibly chokes on something, becomes unresponsive for a few seconds, 
then goes back to the main story list on its own. It is then possible to just 
re-select the game and keep playing.


What version of Frotz are you using?
The latest. Is it 1.7? I think it's 1.7.

What device model are you using (e.g., iPhone 5, iPhone 3GS, iPad 4, iPod
Touch 4th gen.)
iPod Touch 5th gen

What version of iOS?  6.1, 7.0, etc.)
8.3


Please provide any additional information below.
This issue only arose when I updated to iOS 8.3.
```

Original issue reported on code.google.com by `ilprimoc...@gmail.com` on 14 Apr 2015 at 11:20

**Comment by PeterPiers** [comment]
This had been sorted previously, but it's come back again! On iOS 9.3.whatever, the latest version, this exact same issue occurs! Though the lag is smaller, it doesn't become unresponsive for as long.

**Comment by spathiwa** [comment]
This is fixed by rebuilding w/IOS 8 SDK.

**Comment by spathiwa** [comment]
Fixed in 1.8 in App Store.
